docs/migration-config/README.md

```diff
@@ -25,10 +31,20 @@ INDEXER_SUBGRAPHS__ESCROW__QUERY_URL=<escrow subgraph query url>
 INDEXER_SUBGRAPHS__ESCROW__DEPLOYMENT_ID=<escrow deployment id>
 ```
 
-We recommend you migrating overtime to be fully in the configuration and don't need to provide multiple environment variables that can stay in plaintext, this will make your work easier.
+We recommend you migrate over time to be fully in the
```

Comment:
The English is a bit off, but then I also wonder whether this bit is needed at all? Or maybe something simpler like "Users typically have an easier time using the config over environment variables."

Reply:
Weird, I passed it through Grammarly.

Reply:
"Migrating overtime to be fully in the configuration" sounds weird.

Reply:
See my suggestion!